packages/showcases/css/stories/components/navigation/tabs/examples/demo.html

```diff
@@ -1,21 +1,196 @@
 <div class="block">
-  <nav id="demo-tabs-nav" aria-label="Tabs list" class="vtmn-tabs">
-    <ul id="demo-tabs-tablist" role="tablist">
-      <span id="block-focus-trap-start" tabindex="-1"></span>
-      <li id="tab-demo-1" role="tab" aria-selected="false">
-        <a href="" class="selected">Tab Name</a>
+  <nav id="demo-tabs-nav" class="vtmn-tabs">
+    <ul id="demo-tabs-tablist" role="tablist" aria-label="Tabs navigation">
```

Comment:
@GaspardMathon `ul` and `li` being only semantic tags to introduce a list, isn't better to let the `aria-label="Tabs navigation"` on the `nav` tag ? It is the element that the user will reach with tab and screen-reader and thus the element that needs to be announced with an aria-label no ?

Reply:
To go further, you should not use the `<nav>` HTML element here. 

The pattern to follow for this component is [Tabs](https://www.w3.org/WAI/ARIA/apg/patterns/tabpanel/). 

The `<nav>` HTML element must be used only in some specific cases : [Menu et barre de navigation (fr)](https://www.numerique.gouv.fr/publications/rgaa-accessibilite/methode-rgaa/glossaire/#menu-et-barre-de-navigation)

Reply:
You're right, it is not needed for such a component. I will remove it. Thanks for your awareness.